hack/components/components.sh

```diff
@@ -12,59 +11,7 @@ mkdir -p $GOPATH
 export PATH=${GOPATH}/bin:${PATH}
 
 echo 'kubemacpool'
-KUBEMACPOOL_URL=$(cat components.yaml | shyaml get-value components.kubemacpool.url)
-KUBEMACPOOL_COMMIT=$(cat components.yaml | shyaml get-value components.kubemacpool.commit)
-KUBEMACPOOL_REPO=$(echo ${KUBEMACPOOL_URL} | sed 's#https://\(.*\)#\1#')
-KUBEMACPOOL_PATH=${GOPATH}/src/${KUBEMACPOOL_REPO}
-
-echo 'Fetch kubemacpool sources'
-fetch_component ${KUBEMACPOOL_PATH} ${KUBEMACPOOL_URL} ${KUBEMACPOOL_COMMIT}
-
-echo 'Configure kustomize for CNAO templates and save the rendered manifest under CNAO data'
-(
-    cd ${KUBEMACPOOL_PATH}
-    mkdir -p config/cnao
-
-    cat <<EOF > config/cnao/kustomization.yaml
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-namespace: "{{ .Namespace }}"
-bases:
-- ../default
-patchesStrategicMerge:
-- cnao_image_patch.yaml
-EOF
-
-    cat <<EOF > config/cnao/cnao_image_patch.yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: mac-controller-manager
-  namespace: system
-spec:
-  template:
-    spec:
-      containers:
-      - image: "{{ .KubeMacPoolImage }}"
-        imagePullPolicy: "{{ .ImagePullPolicy }}"
-        name: manager
-EOF
-)
-rm -rf data/kubemacpool/*
-(
-    cd $KUBEMACPOOL_PATH
-    kustomize build config/cnao | \
-        sed 's/kubemacpool-system/{{ .Namespace }}/' | \
-        sed 's/RANGE_START: .*/RANGE_START: {{ .RangeStart }}/' | \
-        sed 's/RANGE_END: .*/RANGE_END: {{ .RangeEnd }}/'
-) > data/kubemacpool/kubemacpool.yaml
-
-echo 'Get kubemacpool image name and update it under CNAO'
-KUBEMACPOOL_TAG=$(get_component_tag ${KUBEMACPOOL_PATH})
-KUBEMACPOOL_IMAGE=quay.io/kubevirt/kubemacpool
-KUBEMACPOOL_IMAGE_TAGGED=${KUBEMACPOOL_IMAGE}:${KUBEMACPOOL_TAG}
-sed -i "s#\"${KUBEMACPOOL_IMAGE}:.*\"#\"${KUBEMACPOOL_IMAGE_TAGGED}\"#" pkg/components/components.go
-sed -i "s#\"${KUBEMACPOOL_IMAGE}:.*\"#\"${KUBEMACPOOL_IMAGE_TAGGED}\"#" test/releases/${CNAO_VERSION}.go
+source hack/components/bump_kubemacpool.sh
```

Comment:
Can you just call it? Is there a real need to source it?

Reply:
do you mean to do `./hack/components/bump_kubemacpool.sh`?
I can I think, but what's the difference?

Reply:
wil change